### ISSUE TITLE
Games launch to desktop, but not HMD

### ISSUE THREAD

**Issue by michalowskib** [LibreVR/Revive#545]
I have the same issues as described in #478 and #486
I can start bullet train, Luckys Tale, Robo Recall, SuperHot and few others without problems. Henry, touch turtorial, dreamdesk and some other just won't work. Only black background and sounds in HMD.

1. Revive tab still shows up in the tray, and games are still in there.
2. When you click on a game, you get the circular waiting animation. After a while, you get the in-game sounds in the headset. On the monitor, you'll see that the game has started, a window shows a view from inside the game (as it would normally during a VR game). 

I will try to reinstall Oculus Software, but in case it won't help i wanted to post this here while I'm at work

**Comment by Armada651** [comment]
You're using the SteamVR beta, please revert to the stable version.

**Comment by smeenz** [comment]
I had exactly this after installing the oculus software, and then revive 2 days ago.  Got the spinning ticker in the HMD, and the game launches on the desktop. 

However, all I had to do was reboot the PC, and everything started working after that, so I assume there's some sort of dependency or config setting that only gets set properly after a reboot.

I am not running the steam vr beta.

**Comment by michalowskib** [comment]
In my case, going back from SteamVR beta to stable  solved the problem. I will go back to beta, and if this happens in a game again, i will know what to do ;)

**Comment by Armada651** [comment]
No reason to go back to the beta, it's a known issue.

**Comment by Casyi** [comment]
Hey there, I am having a similar issue. I am trying to launch a game that is a prerelease though, so it's not on the Revive combatibility list. Still hoping that I could try this game using Revive, because the game does launch (not sure if this is an indication that it might work), but it seems that the game loaded on my desktop screen doesn't recognize like my Vive. It looks like the hmd is on the floor in-game. I tried Dreamdeck as well, that works fine. I'm hoping there's a fix that you might know that could solve this.

Also, I am not using the SteamVR beta. Below I've attached my logs. 

[ReviveInjector.txt](https://github.com/LibreVR/Revive/files/914511/ReviveInjector.txt)
[ReviveOverlay.txt](https://github.com/LibreVR/Revive/files/914512/ReviveOverlay.txt)

What OP's issues are, are exactly the same as mine. I have also done as the person above me has stated at 3:

1. Revive tab still shows up in the tray, and my games are still in there.
2. When you click on a game, you get the circular waiting animation. After a while, you get the in-game sounds in the headset. On the monitor, you'll see that the game has started, a window shows a view from inside the game (as it would normally during a VR game). The view does not correspond to the headset orientation, though. Controllers are visible in front of you in this windowed view, and do not respond to movements of the actual Vive controllers.
3. I have uninstalled and reinstalled Oculus, and then reinstalled the latest version of Revive (as of today). Same problem persists.

Hope you have the time to check out if there's a fix for this. Thanks in advance.
